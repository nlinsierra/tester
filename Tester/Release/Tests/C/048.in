368 142 1391 
capital
c
d
e
f
g
h
i
j
ab
bb
cb
db
eb
fb
gb
hb
ib
jb
ac
bc
cc
dc
ec
fc
gc
hc
ic
jc
ad
bd
cd
dd
ed
fd
gd
hd
id
jd
ae
be
ce
de
ee
fe
ge
he
ie
je
af
bf
cf
df
ef
ff
gf
hf
if
jf
ag
bg
cg
dg
eg
fg
gg
hg
ig
jg
ah
bh
ch
dh
eh
fh
gh
hh
ih
jh
ai
bi
ci
di
ei
fi
gi
hi
ii
ji
aj
bj
cj
dj
ej
fj
gj
hj
ij
jj
aab
bab
cab
dab
eab
fab
gab
hab
iab
jab
abb
bbb
cbb
dbb
ebb
fbb
gbb
hbb
ibb
jbb
acb
bcb
ccb
dcb
ecb
fcb
gcb
hcb
icb
jcb
adb
bdb
cdb
ddb
edb
fdb
gdb
hdb
idb
jdb
aeb
beb
cureburg
capital bc 124
capital hc 33
capital ic 89
capital dd 33
capital fd 17
capital ce 83
capital fe 171
capital je 76
capital jh 40
capital ci 72
capital ii 40
capital dj 16
capital jab 104
capital gcb 157
capital adb 44
capital beb 46
capital cureburg 175
c h 165
c cb 158
c fb 4
c bc 89
c id 60
c ce 12
c he 85
c jg 52
c eh 89
c bi 151
c di 144
c ei 30
c hi 47
c ii 79
c hj 74
c ij 23
c bab 120
c dab 144
c gab 111
c ibb 86
c dcb 88
c icb 102
c edb 49
c fdb 108
c cureburg 120
d eb 157
d fb 57
d ac 158
d bd 58
d dd 34
d fd 51
d jd 101
d ae 9
d be 15
d ce 111
d ge 143
d ie 183
d hf 138
d jf 57
d ag 91
d dh 91
d gh 95
d ih 117
d ii 89
d bab 107
d cab 35
d jbb 151
d hcb 138
d gdb 172
d beb 179
e f 73
e h 90
e j 110
e eb 103
e hb 46
e jb 128
e cc 76
e ic 90
e dd 64
e ed 98
e de 44
e bf 115
e cg 129
e fg 79
e eh 172
e gh 142
e hh 48
e gi 39
e ibb 126
e dcb 121
e bdb 165
e cdb 88
e cureburg 146
f g 84
f ab 3
f bb 130
f ib 130
f jb 20
f cc 12
f ae 70
f ce 95
f ef 68
f ig 145
f hh 95
f di 66
f ej 44
f hj 158
f ebb 44
f hbb 51
f fdb 48
f cureburg 23
g i 178
g hb 151
g ib 83
g de 58
g jg 128
g ch 172
g ih 95
g ai 16
g di 184
g cj 5
g ebb 177
h cc 125
h ed 17
h jd 168
h de 142
h ee 74
h ie 71
h af 166
h ff 134
h dg 169
h gh 65
h ih 140
h jh 170
h ci 34
h ei 109
h cj 66
h dj 139
h hj 19
h fbb 21
h gbb 115
h gcb 6
h hcb 46
h beb 171
i bb 86
i cb 47
i hb 165
i ic 102
i cd 21
i jf 27
i di 104
i ji 68
i bj 180
i gj 35
i jj 93
i dab 56
i hab 8
i dbb 117
i fbb 124
i gcb 148
i jdb 121
j ib 66
j ac 22
j hc 153
j ed 78
j hd 172
j id 155
j jd 5
j eg 82
j dj 67
j gj 142
j gcb 71
j hcb 26
ab ac 112
ab cc 50
ab jd 58
ab ge 137
ab bf 19
ab ag 32
ab cg 4
ab fh 172
ab di 110
ab cj 10
ab bab 31
ab bbb 153
ab bcb 156
ab icb 171
bb hb 132
bb cc 36
bb ed 165
bb ff 114
bb gg 33
bb hh 149
bb bi 97
bb ci 7
bb fab 49
bb jab 101
bb hbb 75
bb ccb 108
bb ecb 176
cb ic 30
cb jc 80
cb id 181
cb de 172
cb fe 180
cb ge 104
cb he 153
cb ef 136
cb gg 127
cb ai 106
cb di 40
cb ej 145
cb dbb 19
cb acb 22
db fb 127
db ec 6
db id 162
db ce 158
db je 8
db cf 82
db if 167
db hg 11
db cj 50
db dab 16
db iab 139
db jab 66
db dbb 7
db gbb 8
db ecb 179
db jcb 11
db edb 68
db fdb 134
eb gc 35
eb jc 17
eb bd 141
eb gd 114
eb hd 19
eb de 152
eb fe 32
eb je 73
eb ag 21
eb hg 51
eb eh 79
eb fh 149
eb jh 32
eb ei 174
eb bj 175
eb fj 4
eb eab 141
eb gab 184
eb hab 174
eb iab 148
eb acb 105
eb jcb 78
eb ddb 75
eb edb 66
eb fdb 85
eb aeb 44
fb ib 101
fb ac 20
fb ec 74
fb fd 25
fb jd 128
fb be 16
fb ge 83
fb fg 122
fb jg 130
fb eh 152
fb ih 69
fb jh 48
fb gj 139
fb gab 125
fb ebb 39
fb fbb 71
fb acb 140
fb edb 154
gb ib 149
gb bc 158
gb dc 41
gb fd 53
gb ee 116
gb he 82
gb jg 50
gb jh 142
gb ej 66
gb gj 138
gb ij 60
gb dab 155
gb eab 92
gb gbb 122
gb icb 31
gb adb 133
gb ddb 168
gb gdb 120
gb jdb 8
hb ib 131
hb cc 77
hb cd 23
hb fe 19
hb ge 40
hb df 3
hb gf 172
hb hf 36
hb ig 10
hb bh 15
hb dh 133
hb fh 51
hb gh 33
hb ci 176
hb gi 81
hb eab 16
hb fab 147
hb abb 85
hb jcb 48
hb ddb 75
ib jb 55
ib fc 40
ib fd 15
ib be 144
ib ee 48
ib ie 143
ib df 20
ib ig 152
ib eh 62
ib ih 96
ib gi 42
ib hi 38
ib ji 99
ib aab 116
ib dbb 21
ib bcb 162
ib gdb 170
ib hdb 102
ib beb 110
jb ee 170
jb df 99
jb cg 170
jb bh 16
jb dh 142
jb di 155
jb gi 89
jb aj 63
jb fj 2
jb aab 164
jb jab 18
jb ibb 109
jb ccb 184
jb edb 142
jb aeb 142
ac de 71
ac ie 139
ac je 110
ac cf 59
ac df 39
ac ff 59
ac if 38
ac fg 138
ac ih 115
ac bi 74
ac aj 23
ac cab 121
ac dab 112
ac gcb 162
ac cdb 62
ac hdb 103
ac beb 124
bc cc 170
bc dc 171
bc dd 58
bc ef 171
bc ff 70
bc cg 87
bc ig 145
bc jg 52
bc ah 160
bc eh 88
bc jh 96
bc gi 89
bc bj 112
bc bab 90
bc cab 52
bc jab 168
bc acb 148
bc dcb 42
bc fcb 118
bc gcb 2
bc icb 15
bc cdb 21
cc dd 20
cc fd 120
cc gd 130
cc he 64
cc je 2
cc if 143
cc cg 92
cc dg 159
cc fg 153
cc jg 174
cc aj 94
cc ej 87
cc dbb 123
cc hdb 57
dc ec 124
dc jc 177
dc gd 90
dc ae 171
dc fe 54
dc ge 2
dc if 174
dc ig 32
dc dj 5
dc gj 30
dc hab 16
dc cbb 107
dc fbb 92
dc jbb 182
dc bcb 147
dc jcb 182
ec jd 147
ec de 69
ec cf 55
ec ff 179
ec eg 24
ec hg 148
ec ah 9
ec dh 39
ec jh 106
ec di 106
ec ei 17
ec hi 12
ec ii 174
ec ji 80
ec fj 127
ec cab 138
ec fab 81
ec jbb 111
ec hcb 5
ec jcb 93
ec gdb 97
fc ic 139
fc ad 180
fc gd 112
fc he 85
fc ie 106
fc df 9
fc dg 22
fc eg 122
fc ei 65
fc cj 148
fc gbb 141
fc hbb 1
fc ddb 140
fc edb 142
fc gdb 17
gc jc 26
gc cd 88
gc dd 47
gc fe 130
gc ie 17
gc df 95
gc ef 35
gc ff 41
gc jf 91
gc ag 84
gc cg 105
gc bh 70
gc fh 154
gc gh 68
gc ci 181
gc hi 81
gc gab 156
gc dbb 46
gc fbb 61
gc ibb 127
gc dcb 2
gc fdb 176
hc ae 173
hc ge 51
hc cf 94
hc hf 82
hc if 50
hc bg 184
hc eg 14
hc fg 32
hc ah 162
hc fh 107
hc ci 177
hc dbb 119
hc ccb 74
hc ddb 3
ic hd 99
ic bf 49
ic jf 178
ic bg 48
ic ih 92
ic ai 31
ic dab 103
ic cbb 10
ic ccb 98
ic hcb 21
ic ddb 153
ic edb 180
ic beb 158
ic cureburg 97
jc ed 142
jc gd 147
jc be 39
jc af 106
jc jg 128
jc di 159
jc ei 110
jc gi 144
jc fab 49
jc jbb 118
jc bcb 88
jc bdb 158
jc ddb 36
jc fdb 106
jc gdb 109
jc aeb 115
ad cf 169
ad hh 135
ad jh 16
ad fj 20
ad eab 69
ad gab 81
ad abb 55
ad jbb 108
bd cd 138
bd hd 37
bd id 136
bd ae 154
bd ce 123
bd fg 104
bd bh 28
bd jh 26
bd ji 95
bd ij 138
bd abb 124
bd ebb 130
bd ecb 101
bd adb 52
cd dd 122
cd jd 141
cd be 61
cd ce 66
cd de 122
cd fe 2
cd he 83
cd bf 181
cd eh 40
cd ai 184
cd cj 165
cd dj 98
cd fcb 20
dd fd 21
dd id 58
dd fe 79
dd jf 32
dd ag 65
dd ch 71
dd fh 163
dd bj 138
dd hj 28
dd bbb 42
dd ecb 74
dd idb 75
ed jd 29
ed df 183
ed bg 184
ed jg 111
ed dh 71
ed eh 100
ed ih 99
ed gi 144
ed dj 110
ed iab 171
ed cbb 23
ed dbb 172
ed hbb 36
ed bcb 160
ed gcb 117
fd hd 130
fd bh 18
fd eh 50
fd fi 100
fd cj 166
fd gab 6
fd gbb 120
fd acb 78
fd fdb 108
gd id 8
gd ef 12
gd jf 122
gd cg 98
gd dg 98
gd hg 67
gd jh 88
gd ei 91
gd fi 138
gd bj 11
gd ij 39
gd jj 158
gd aab 126
gd fbb 56
gd hcb 147
gd bdb 154
gd edb 40
gd hdb 66
gd aeb 104
hd je 51
hd hf 60
hd jf 113
hd bi 125
hd ji 41
hd aj 64
hd ej 58
hd gj 67
hd ij 131
hd cab 136
hd iab 136
hd fbb 173
hd hbb 108
hd acb 139
hd dcb 47
hd gcb 154
hd cdb 17
hd ddb 150
hd idb 38
id af 87
id bf 43
id jf 150
id hg 25
id ig 51
id jg 83
id ci 122
id cj 114
id cbb 73
id edb 109
id hdb 128
id idb 87
id aeb 45
id beb 134
jd be 24
jd ge 20
jd ff 45
jd hg 153
jd bh 169
jd hh 36
jd aj 104
jd fj 34
jd dab 161
jd hab 15
jd abb 135
jd bbb 85
jd cbb 43
jd dcb 24
jd aeb 164
ae cf 100
ae if 111
ae bg 58
ae dg 89
ae eg 45
ae ch 58
ae fh 31
ae jh 118
ae bi 62
ae bj 100
ae cj 6
ae dj 147
ae bbb 176
ae ebb 151
ae jbb 151
ae bcb 38
ae jcb 61
ae ddb 108
ae cureburg 98
be ce 151
be de 148
be je 15
be ef 127
be ah 86
be gh 116
be ei 153
be ii 5
be fj 177
be iab 148
be dbb 7
be acb 15
be dcb 39
be fcb 34
be hcb 139
be beb 9
ce ee 156
ce df 139
ce ef 148
ce if 147
ce ag 42
ce cg 61
ce eg 184
ce gg 32
ce ji 27
ce aj 137
ce dj 52
ce fj 127
ce gj 36
ce aab 78
ce cbb 128
ce hbb 114
ce hcb 147
ce edb 154
ce aeb 165
de ag 25
de dg 141
de ig 82
de hh 92
de ai 14
de cj 145
de hj 22
de ij 32
de jab 70
de fcb 149
de gcb 143
de hcb 84
de icb 64
de edb 37
de fdb 173
de beb 68
ee ie 106
ee bg 44
ee cg 78
ee gg 130
ee dh 82
ee aj 45
ee jj 91
ee dab 37
ee bbb 17
ee hbb 139
ee ibb 53
ee jbb 174
ee bcb 5
ee gcb 112
ee icb 163
ee edb 104
ee jdb 47
fe he 1
fe ie 97
fe eg 64
fe hg 40
fe ig 121
fe bh 32
fe ch 98
fe ih 41
fe di 119
fe dj 177
fe fj 62
fe fab 76
fe gab 60
fe jab 76
fe cbb 163
fe dbb 152
fe ccb 49
fe gdb 75
ge je 127
ge cf 39
ge gf 182
ge ag 123
ge ch 110
ge fh 56
ge gh 18
ge ih 92
ge ej 57
ge fj 156
ge adb 17
he bf 112
he if 179
he bh 26
he hh 165
he ai 137
he hi 178
he cbb 175
he fcb 133
he bdb 47
he ddb 120
he edb 6
he idb 68
ie af 20
ie ff 75
ie hf 83
ie jf 77
ie ih 64
ie fi 21
ie aj 32
ie gj 155
ie eab 73
ie iab 89
ie cbb 153
ie hbb 55
ie ibb 46
ie ccb 62
ie fcb 116
ie ddb 76
je cf 184
je gf 31
je jf 7
je jg 59
je eh 125
je fh 58
je cj 90
je ecb 131
je fcb 82
je edb 183
je cureburg 59
af dh 67
af aj 27
af jj 81
af jab 22
af hbb 152
af bdb 74
af ddb 167
af jdb 32
af aeb 138
bf df 57
bf if 62
bf bg 179
bf eg 139
bf ih 157
bf gi 3
bf hj 165
bf aab 53
bf eab 54
bf abb 72
bf ibb 91
bf hdb 73
bf jdb 7
bf cureburg 51
cf jf 133
cf eg 86
cf jg 149
cf ei 38
cf ii 16
cf abb 164
cf bbb 151
cf ibb 110
cf acb 143
cf cdb 150
cf fdb 138
cf hdb 127
cf aeb 124
df if 72
df fg 142
df jg 118
df hh 175
df ih 86
df ai 12
df hi 119
df ej 130
df jj 40
df hab 176
df dcb 113
df fcb 180
df fdb 81
ef ff 22
ef gf 43
ef bh 19
ef jj 174
ef cab 5
ef gab 14
ef fbb 37
ef hbb 46
ef idb 26
ef beb 124
ff bg 166
ff hg 63
ff ig 54
ff ch 103
ff ai 153
ff fi 91
ff jj 91
ff jab 159
ff jbb 4
ff acb 166
ff ecb 83
ff hcb 101
ff cdb 5
ff gdb 44
ff beb 137
gf jf 55
gf bg 183
gf eg 157
gf ih 27
gf jh 79
gf hi 8
gf dab 180
gf fab 158
gf jab 65
gf ecb 121
gf cdb 135
gf ddb 28
gf beb 45
hf gg 143
hf hg 105
hf ei 44
hf ji 92
hf bj 150
hf ej 5
hf iab 127
hf abb 128
hf fbb 86
hf gbb 158
hf ddb 99
hf edb 69
if ag 103
if eg 65
if ii 87
if cab 31
if abb 140
if bbb 84
if gbb 71
if jcb 66
if fdb 27
jf hg 24
jf gh 117
jf fi 154
jf ji 89
jf aj 178
jf cj 75
jf bab 11
jf cab 133
jf hab 65
jf edb 133
jf gdb 132
ag ch 79
ag ai 145
ag di 10
ag hi 70
ag ji 124
ag bj 33
ag fab 137
ag icb 149
ag jcb 107
ag fdb 150
ag hdb 96
bg jg 82
bg ch 38
bg gh 95
bg ih 175
bg ej 157
bg gab 23
bg hab 25
bg icb 149
cg dg 15
cg gg 6
cg ig 48
cg dh 103
cg gh 147
cg ci 114
cg di 99
cg hi 138
cg cj 4
cg fj 13
cg gj 75
cg hj 101
cg dbb 48
cg ebb 141
cg jbb 43
cg icb 96
cg jcb 71
cg adb 140
cg gdb 2
cg idb 10
cg jdb 175
dg gg 128
dg cj 167
dg gj 9
dg aab 108
dg ccb 152
dg hcb 111
dg bdb 172
eg fg 139
eg gg 7
eg ig 49
eg fh 161
eg ci 158
eg cj 103
eg fab 38
eg cdb 48
eg edb 145
eg idb 47
eg jdb 62
eg aeb 21
eg beb 157
eg cureburg 133
fg hg 1
fg bh 3
fg ch 37
fg eh 180
fg jh 176
fg ai 38
fg bi 133
fg ci 43
fg bj 4
fg jj 129
fg aab 90
fg dab 15
fg ibb 121
fg acb 168
fg dcb 155
fg bdb 136
fg edb 16
fg fdb 71
fg cureburg 16
gg hh 123
gg gi 90
gg hi 177
gg ej 20
gg fab 13
gg acb 151
gg ecb 71
gg fcb 18
gg bdb 91
hg aj 84
hg cj 159
hg bab 62
hg eab 135
hg ebb 91
hg icb 96
hg gdb 25
ig fh 103
ig ii 139
ig gab 178
ig ecb 49
ig fdb 52
ig gdb 16
ig aeb 3
jg jh 135
jg ii 180
jg dj 69
jg ij 55
jg dab 133
jg iab 168
jg cbb 76
jg acb 109
jg bdb 164
ah aj 175
ah hj 165
ah fab 20
ah acb 28
ah hcb 133
ah idb 158
ah jdb 173
ah beb 58
bh ch 84
bh eh 32
bh ih 58
bh ci 121
bh ii 96
bh aj 179
bh cj 50
bh ej 16
bh ebb 75
bh fbb 53
bh icb 137
bh cdb 133
bh beb 167
ch aj 19
ch cj 157
ch aab 183
ch bab 36
ch cab 112
ch eab 27
ch hab 116
ch fbb 107
ch hcb 139
ch gdb 71
ch jdb 163
dh ih 148
dh gj 131
dh aab 111
dh cab 4
dh dab 9
dh fab 34
dh iab 105
dh ebb 50
dh hbb 18
dh bdb 20
dh fdb 58
eh aj 81
eh bj 96
eh gj 104
eh dab 178
eh hab 16
eh abb 118
eh ibb 116
eh gcb 163
eh gdb 21
eh aeb 175
eh beb 8
eh cureburg 166
fh di 19
fh ii 112
fh aab 16
fh fab 82
fh cbb 38
fh fbb 57
fh jdb 177
gh ih 136
gh bi 43
gh gi 153
gh hi 55
gh ej 74
gh iab 156
gh fbb 61
gh bcb 91
gh ccb 102
gh ecb 63
gh gcb 62
gh hdb 114
gh beb 75
hh ei 24
hh hi 35
hh jab 22
hh abb 61
hh fbb 65
hh edb 9
ih ij 167
ih eab 68
ih jab 183
ih cbb 157
ih hbb 116
ih ibb 24
jh gi 183
jh dab 3
jh iab 91
jh gdb 45
ai fj 93
ai iab 125
ai jab 90
ai dbb 6
ai fbb 36
ai bdb 72
ai fdb 30
ai aeb 15
bi di 59
bi ej 147
bi fj 86
bi fab 147
bi hcb 96
bi beb 132
ci dj 15
ci aab 162
ci eab 57
ci ebb 120
ci gbb 174
ci icb 107
ci ddb 15
di ei 171
di aj 154
di dj 157
di ej 114
di fab 137
di iab 2
di cdb 72
di aeb 24
ei aj 89
ei cj 62
ei fj 4
ei dab 38
ei gab 143
ei fbb 139
ei gbb 46
ei adb 179
ei cdb 120
ei ddb 8
ei edb 181
ei fdb 105
ei aeb 27
fi hi 108
fi aj 107
fi cj 7
fi ebb 140
fi bcb 42
fi aeb 90
gi ii 85
gi cj 47
gi ebb 125
gi jbb 138
gi hcb 56
gi idb 40
hi ii 106
hi ji 140
hi dj 26
hi ej 134
hi gj 74
hi jj 115
hi dab 110
hi hbb 3
hi cdb 17
ii eab 70
ii icb 64
ii ddb 35
ji aj 20
ji bab 7
ji abb 180
ji dbb 36
ji hbb 52
ji bcb 182
ji hdb 150
ji cureburg 158
aj cj 26
aj gab 158
aj hab 62
aj jab 29
aj cdb 161
aj hdb 144
bj eab 6
bj gab 112
bj bbb 128
bj edb 15
bj beb 35
cj bab 139
cj gab 132
cj abb 148
cj cbb 99
cj fbb 63
cj dcb 161
cj adb 62
cj gdb 21
dj fbb 42
dj gbb 74
dj hbb 116
dj cdb 62
dj aeb 143
dj cureburg 120
ej ij 29
ej dab 136
ej bbb 10
ej gbb 111
ej ibb 90
ej ecb 133
ej hdb 26
fj jj 140
fj iab 94
fj fcb 175
fj jcb 80
fj gdb 57
fj hdb 27
fj aeb 133
gj aab 4
gj bab 14
gj cbb 83
gj dbb 62
gj acb 143
gj ecb 168
gj icb 34
gj hdb 167
hj jj 46
hj cab 6
hj hab 172
hj jab 76
hj bbb 88
hj cbb 95
hj acb 80
hj ecb 24
hj hcb 85
hj bdb 109
hj edb 77
ij bab 33
ij acb 93
ij dcb 1
ij edb 93
ij idb 174
ij cureburg 7
jj ibb 127
jj acb 30
jj fcb 150
jj cdb 105
jj ddb 179
jj fdb 127
jj jdb 93
aab gab 21
aab bbb 1
aab ebb 110
aab ccb 125
aab dcb 128
aab gcb 138
aab hcb 146
aab bdb 31
aab hdb 141
aab cureburg 73
bab eab 76
bab jab 171
bab hbb 75
bab acb 171
bab cureburg 77
cab fbb 16
cab gcb 7
cab hcb 4
cab adb 183
cab hdb 82
cab aeb 88
dab bcb 68
dab jcb 11
dab bdb 147
eab dbb 52
eab gbb 177
eab dcb 112
eab adb 26
eab bdb 176
eab hdb 18
fab iab 107
fab ibb 122
fab hcb 105
fab edb 73
gab hab 108
gab ecb 147
gab icb 82
gab ddb 158
hab hbb 181
hab ccb 114
hab ddb 42
hab hdb 138
iab bbb 20
iab jbb 85
iab cdb 121
jab dbb 102
jab fbb 56
jab hbb 86
jab hcb 54
jab icb 158
jab hdb 116
abb dbb 80
abb hbb 55
abb hcb 174
abb icb 102
abb fdb 183
bbb hcb 88
bbb bdb 149
bbb cdb 28
bbb gdb 82
cbb fbb 104
cbb ibb 153
cbb aeb 64
cbb cureburg 21
dbb acb 51
dbb jcb 95
dbb jdb 175
dbb cureburg 67
ebb gbb 184
ebb ccb 56
fbb ibb 56
fbb jbb 12
fbb adb 55
fbb edb 146
gbb edb 45
gbb beb 29
hbb dcb 143
hbb bdb 65
hbb hdb 89
ibb cdb 14
jbb hcb 94
jbb jdb 89
acb ecb 184
acb bdb 10
acb edb 79
acb jdb 4
bcb fcb 9
bcb ddb 94
bcb edb 144
ccb ecb 118
ccb fcb 98
dcb icb 70
dcb fdb 130
dcb cureburg 142
ecb cdb 25
fcb hcb 98
fcb cdb 102
fcb ddb 8
fcb idb 167
fcb beb 144
gcb hdb 23
hcb aeb 43
hcb beb 150
hcb cureburg 77
icb adb 150
icb edb 35
jcb fdb 147
adb fdb 98
ddb fdb 29
ddb aeb 1
ddb cureburg 108
edb beb 25
edb cureburg 19
idb aeb 13
