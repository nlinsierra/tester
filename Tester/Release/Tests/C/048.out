YES
capital
fd
bh
fg
cureburg
